packages/clay-icon/src/ClayIcon.tsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import * as React from 'react';
+import classNames from 'classnames';
+import svg4everybody from 'svg4everybody';
```

Comment:
I'm not sure if this package should be here, since we already have it in the Portal.

Reply:
@jbalsas do you think this package should be included to Icon? Other possibilities would be remove it entirely or add a config to enable it.

Reply:
Based on https://github.com/jonathantneal/svg4everybody/blob/master/dist/svg4everybody.js#L99, I think a config to enable it should be fine, otherwise we risk having multiple intervals running and making requests if someone else initializes it on their own.

Other option would be to investigate alternative ways to incorporate the svg spritemap to avoid this issue completely, which could be an unexpected and welcomed outcome ;)